### DISCUSSION TITLE
Why not support iOS 10?

### DISCUSSION THREAD

**Discussion by shannonchou** [utmapp/UTM#3501] [UTM Help (iOS)] [answered]
What is the reason that UTM not support iOS 10?
1) System limitation that we can not overcome.
2) We use a lot of iOS 11 only apis for convenience.

If it is reason 2 I may want to spend some time to make it be compatible with iOS 10. Because I want to run UTM on my iPhone 5c (iOS 10.3.3).

**Comment by osy** [comment, marked as answer]
You need ARM64. The limitation isn’t iOS 11. It’s that iOS 11 is the lowest iOS that old ARM64 devices still can run. Devices that cannot run iOS 11 are ARM32 which means they can’t run QEMU.

**Reply by shannonchou** [reply]
👍  Super clear.
